### PR TITLE
bowang-added two new components: popover & speeddial

### PR DESCRIPTION
**Pull request by bowang-ynd**
Add two new components to dash-mui

working towards #1

### CODE REVIEW

File: src/lib/components/CustomSpeedDial.react.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+import SpeedDialIcon from '@mui/material/SpeedDialIcon';
+import SpeedDialAction from '@mui/material/SpeedDialAction';
+import SaveIcon from '@mui/icons-material/Save';
+import PrintIcon from '@mui/icons-material/Print';
+import ShareIcon from '@mui/icons-material/Share';
+
+/**
+ * Icon mapping for SpeedDial actions.
+ */
+const iconMap = {
```
Comment (LatentDream) [inline src/lib/components/CustomSpeedDial.react.js:15]: The user can only use those 3 icons. Is there a way to have more ?
Like all for the one from `@mui/icons-material/*`

File: demo/pages/popover.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+dash.register_page(__name__)
+
+button_popover_props = {
+    "id": "button-popover",
+    "content": "This is the content of the Popover opened by button click.",
+    "anchorOrigin": {"vertical": "bottom", "horizontal": "left"},
+}
+
+layout = html.Div(
+    mui.Grid(
```
Comment (LatentDream) [inline demo/pages/popover.py:16]: Same for this one: A lot of white space, and the code is cramp in the corner:
image

**Review by LatentDream — CHANGES_REQUESTED** [review]

**Comment by bowang-ynd** [comment]
@LatentDream I have changed the code example size and now the popover and speed dial should take full size of the space now. I also included all icons from the @mui/icons-material now

**Comment by edublancas** [comment]
@LatentDream please re review

File: demo/pages/speedDial.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import dash_material_ui as mui
+import dash
+from dash import html, Input, Output, callback
+from util import create_component_item
+
+dash.register_page(__name__)
+
+speed_dial_actions = [
+    {"id": "save-action", "icon": "Save", "tooltip": "Save"},
```
Comment (LatentDream) [inline demo/pages/speedDial.py:9]:
```suggestion
speed_dial_actions = [
    # Material UI icons can be used by their names
    # Browse available icons at: https://mui.com/material-ui/icons
    {"id": "save-action", "icon": "Save", "tooltip": "Save"},
```

**Review by LatentDream — APPROVED** [review]
I'm suggesting adding a comment about where the user can check the all available icon. Otherwise, Looking good! and Tested on my side

File: demo/pages/speedDial.py (hunk 3 above)
Comment (edublancas) [reply demo/pages/speedDial.py:9]: @bowang-ynd please apply this change

**Comment by bowang-ynd** [comment]
@edublancas @LatentDream I have committed the changes to add a quick comment for users to look for icons doc and their names.

**Pull request merged**